parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/AbstractParallelEoSStreamProcessor.java

```diff
@@ -66,6 +64,14 @@ public abstract class AbstractParallelEoSStreamProcessor<K, V> implements Parall
      */
     private static final String MDC_WORK_CONTAINER_DESCRIPTOR = "offset";
 
+    /**
+     * Timeout used by various subsystems(BrokerPoller, Consumer) during shut down.
+     */
+    public static final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(30);
```

Comment:
Feels like something that should be loaded as a config? If it's shared in multiple classes it feels a bit weird to be controlled here. Should it be in `ParallelConsumerOptions`?

Reply:
Not really sure - it was hard coded in the DrainMode enum but i've added config for shutdown and drain operations in the ParallelConsumerOptions. 
Still need this default timeout for other operations that were using it... i dont really want to make it configurable just yet as its used for commit response blocking timeout and for poller subsystem shutdown.